src/pages/index.js

```diff
@@ -12,56 +11,204 @@ import {
+        profileInfo.setUserInfo(r);
+      })
+      .finally(() => {
+        renderLoading(false, profileEditForm, "Save");
+      });
+  },
+  handleClearError: () => {
+    profileFormValidator.clearError();
+  },
+  renderLoading: () => {
+    renderLoading(true, profileEditForm);
   },
 });
 profilePopup.setEventListeners();
 profileButtonEdit.addEventListener("click", () => {
-  fillProfileForm(profileEddited.getUserInfo());
+  fillProfileForm(profileInfo.getUserInfo());
   profilePopup.open();
 });
 
-const cardsPopup = new PopupWithForm(cardsModal, {
-  handleFormSubmit: (obj) => {
-    const cardElement = createCard(obj);
-    cardsSection.addItem(cardElement);
+//
+
+export const api = new Api(
+  {
+    baseUrl: "https://around-api.en.tripleten-services.com/v1",
+    headers: {
+      authorization: "f01264e8-2101-4ab7-b120-b09b55e63681",
+      "Content-Type": "application/json",
+    },
+  },
+  checkResponse
+);
+
+/*check:
+const newCardsSection = new Section(
+  {
+    data: "",
+    renderer: (item) => {
+      const cardElement = createCard(item);
+      cardList.append(cardElement);
+    },
+  },
+  cardList
+);
+*/
+api
+  .returnUserInfoAndCards()
+  .then((res) => {
+    profileInfo.setUserInfo(res[0]);
+
+    const newCardsSection = new Section(
+      {
+        data: res[1],
+        renderer: (item) => {
+          const cardElement = createCard(item);
+          cardList.append(cardElement);
+        },
+      },
+      cardList
+    );
+    newCardsSection.renderItems();
+  })
+  .catch((err) => console.error(err));
+
+const avatarPopup = new PopupWithForm(
+  propicModal,
+  {
+    handleFormSubmit: (obj) => {
+      api
+        .updateProPic(obj.link)
+        .then((r) => {
+          avatarPopup.close();
+          propic.src = obj.link;
+        })
+        .finally(() => {
+          renderLoading(false, propicForm, "Save");
+        });
+    },
+    handleClearError: () => {
+      propicFormValidator.clearError();
+    },
+    renderLoading: () => {
+      renderLoading(true, propicForm);
+    },
+  },
+  propicFormValidator
+);
+avatarPopup.setEventListeners();
+
+propicButtonEdit.addEventListener("click", () => {
+  avatarPopup.open();
+});
+
+export const deletePopup = new PopupWithConfirmation(deleteModal, {
+  handleFormSubmit: (cardElement, objId) => {
+    api
+      .deleteCard(objId)
+      .then((r) => {
+        cardElement.remove();
+        deletePopup.close();
+      })
+      .finally(() => {
+        renderLoading(false, deleteForm, "Yes");
+      });
+  },
+  renderLoading: () => {
+    renderLoading(true, deleteForm);
   },
 });
+deletePopup.setEventListener();
+
+const cardsPopup = new PopupWithForm(
+  cardsModal,
+  {
+    handleFormSubmit: (obj) => {
+      api
+        .addNewCard(obj)
+        .then((r) => {
+          cardsPopup.close();
+          /*
+      const cardElement = createCard(r);
+      cardList.prepend(cardElement);
+      */
+          const addedCS = new Section(
```

Comment:
Only one instance of class `Section` should be created in the project because there is only **one** section for the cards.